### PR TITLE
test: split integration test into each command

### PR DESCRIPTION
**Pull request by ryota-sakamoto**
*Issue #, if available:*

N/A

*Description of changes:*

We need to add integration test for each command to check compatibility. So I split integration test into each command.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/common.rs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (StoneDot) [inline tests/common.rs:1]: I think those tests do not intend to be against the module commonly used. I think `auxiliary.rs` is a more appropriate name.

**Review by StoneDot — COMMENTED** [review]
I commented on some points regarding naming and structure. Could you please check this?

**Comment by ryota-sakamoto** [comment]
I have fixed some points

**Comment by StoneDot** [comment]
It looks good to me.

**Review by StoneDot — APPROVED** [review]

**Pull request merged**